src/main/scala/shade/memcached/internals/Status.scala

```diff
@@ -1,5 +1,7 @@
 package shade.memcached.internals
 
+import net.spy.memcached.ops.OperationStatus
+
 sealed trait Status
```

Comment:
Got it. I've made this change.

As a side note, since Spymemcached's `OperationStatus` isn't `Serializable`, I've changed `UnhandledStatus` to only hold values we extract from `OperationStatus`.